bindata/network/openshift-sdn/alert-rules.yaml

```diff
@@ -42,13 +49,18 @@ spec:
     - alert: ClusterProxyApplySlow
       annotations:
         summary: The cluster is taking too long, on average, to apply kubernetes service rules to iptables.
+        description: |
+          Cluster network configuration is taking too long to be applied to the cluster. This results in services
+          taking a longer time to provision on the cluster.
```

Comment:
The text here should be mostly parallel to `NodeProxyApplySlow`, but emphasize that it's talking about the cluster as a whole rather than a single bad node.